api/src/pcapi/routes/backoffice/venues/blueprint.py

```diff
@@ -225,13 +225,11 @@ def get_venue(venue_id: int) -> sa.engine.Row:
 
 def render_venue_details(venue_row: sa.engine.Row, edit_venue_form: forms.EditVirtualVenueForm | None = None) -> str:
     venue = venue_row.Venue
-    region = ""
-    if venue.offererAddress:
-        region = (
-            regions_utils.get_region_name_from_postal_code(venue.offererAddress.address.postalCode)
-            if venue.offererAddress.address.postalCode
-            else ""
-        )
+    region = (
+        regions_utils.get_region_name_from_postal_code(venue.offererAddress.address.postalCode)
+        if venue.offererAddress.address.postalCode
+        else ""
+    )
```

Comment:
`Address.postalCode` étant `nullable=False`, on devrait même pouvoir simplifier : 
```suggestion
    region = regions_utils.get_region_name_from_postal_code(venue.offererAddress.address.postalCode)
```